test/unit/parser.test.js

```diff
@@ -45,10 +45,11 @@ test('includeFile replaces <include> with <div>', async () => {
 
   const expected = [
     '# Index',
-    `<div cwf="${indexPath}" include-path="${includePath}">`,
+    `<div cwf="${indexPath}" include-path="${includePath}">`
+    + `<div data-included-from="${includePath}" cwf="${includePath}">`,
```

Comment:
Is there a reason why we couldn't piggyback the `cwf` attribute for our implementation of relative paths?

Reply:
I believe we erase this cwf after we do `resolveBaseUrl`, right before we output the file but before we process any post-process plugins. 
I didn't use the existing cwf attribute mainly because I'm not sure what the cwf contains. In most cases, it looks the same as what I put inside data-included-from, but I'm not too sure.
Using cwf also means that I would have to erase the cwf tag at a later stage, and I'm not sure if I will break any existing implementation by doing that. I don't think so, but I was thinking it might be safer to add data-included-from instead.
As a bonus, being standalone, we know that it would be safe to persist this information till much later in the workflow, at least till the post-process plugins, but potentially till the very end of our process if we want to.

Reply:
I am happy with the current implementation, no need to change. Cleanups can be done in the future once we streamlined the Parser in #810.